scripts/check_reproducible_builds.py

```diff
@@ -0,0 +1,75 @@
+import os
```

Comment:
- header with python shebang
- license
- copyright
- quick into into the script, what does it do, how to use it.